cfgov/unprocessed/css/organisms/video-player.less

```
@@ -70,12 +70,14 @@
 .o-video-player_play-btn {
```

Looks like we need to add this for the button to be the correct size when it's an `a` element.

```suggestion
.o-video-player_play-btn {
    box-sizing: border-box;
```

---

Not sure what's up with it showing a deleted space before the selector. Maybe don't use the UI to accept this suggestion and do it manually.

---

Fixed